lib/autoupdate/agent/setup.go

```diff
@@ -370,6 +364,13 @@ func writeSystemTemplate(path, t string, values any) error {
 		"replace": func(s, old, new string) string {
 			return strings.ReplaceAll(s, old, new)
 		},
+		"escape": func(s string) string {
+			replacer := strings.NewReplacer(
+				`"`, `\"`,
+				`\`, `\\`,
+			)
+			return replacer.Replace(s)
+		},
```

Comment:
Can you add a comment explaining what this function does, and link to the relevant systemd unit docs?